resources/views/layout.blade.php

```diff
@@ -55,42 +55,53 @@
                     <li class="dropdown dropdown-large">
-                        <a href="#" class="dropdown-toggle" data-toggle="dropdown">Community <b class="caret"></b></a>
+                        {!! link_to_route("mship.feedback.new", "Feedback") !!}
+                    </li>
+                </ul>
+
+                <ul class="nav navbar-nav navcustom">
+                    <li class="dropdown dropdown-large">
+                        <a href="#" class="dropdown-toggle" data-toggle="dropdown">Resources <b class="caret"></b></a>
                         <ul class="dropdown-menu dropdown-menu-large row mainmenu_dropdown">
-                            <li class="col-sm-6">
+                            <li class="col-sm-12">
                                 <ul>
-                                    <li class="dropdown-header">Division Membership</li>
-                                    <li>{!! link_to_route("visiting.landing", "Visiting the UK") !!}</li>
-                                    <li>{!! link_to_route("visiting.landing", "Transfer to the UK") !!}</li>
-                                    <li>{!! link_to_route("visiting.landing", "Leaving the UK") !!}</li>
-                                    <li class="divider"></li>
-                                    <li class="dropdown-header">Network Data</li>
-                                    <li>{!! link_to_route("networkdata.online", "Who's online") !!}</li>
+                                    <li>{{ HTML::link('https://vatsim.uk/', 'VATSIM UK Homepage', array("target"=>"_blank")) }}</li>
+                                    <li>{{ HTML::link('https://cts.vatsim.uk/', 'Central Training System', array("target"=>"_blank")) }}</li>
+                                    <li>{!! link_to_route("networkdata.online", "Who's Online") !!}</li>
+                                    <li>{{ HTML::link('http://www.nats-uk.ead-it.com/public/index.php%3Foption=com_content&task=blogcategory&id=6&Itemid=13.html', 'UK Charts', array("target"=>"_blank")) }}</li>
                                 </ul>
                             </li>
-                            <li class="col-sm-6">
+                        </ul>
+                    </li>
+                </ul>
+
+                <ul class="nav navbar-nav navcustom">
+                    <li class="dropdown dropdown-large">
+                        <a href="#" class="dropdown-toggle" data-toggle="dropdown">Community <b class="caret"></b></a>
+                        <ul class="dropdown-menu dropdown-menu-large row mainmenu_dropdown">
+                            <li class="col-sm-12">
                                 <ul>
+                                    <li class="dropdown-header">Membership</li>
+                                    <li>{!! link_to_route("visiting.landing", "Visit the UK") !!}</li>
+                                    <li>{!! link_to_route("visiting.landing", "Transfer to the UK") !!}</li>
```

Comment:
Do you think it would be working combining these into one Visit/Transfer link seeing as they go to the same place?